libs/angular-components/community/components/form/file-dropzone/file-dropzone.component.ts

```diff
@@ -0,0 +1,352 @@
+import {
+  Component,
+  computed,
+  EventEmitter,
+  HostListener,
+  inject,
+  input,
+  Output,
+  signal,
+  viewChild,
+  model,
+  ElementRef,
+  effect,
+  OnInit,
+} from "@angular/core";
+import { ControlValueAccessor, ReactiveFormsModule } from "@angular/forms";
+import {
+  ClosingButtonComponent,
+  FeedbackTextComponent,
+  IconComponent,
+  TooltipComponent,
+  TooltipTriggerComponent,
+  TooltipContentComponent,
+  InfoButtonComponent,
+} from "@tehik-ee/tedi-angular/tedi";
+import { CardComponent, CardContentComponent } from "../../cards";
+import { VerticalSpacingDirective } from "tedi/directives";
+import { TediTranslationPipe, TediTranslationService } from "tedi/services";
+import {
+  DropzoneValidatorFunction,
+  FeedbackTextProps,
+  FileDropzone,
+  FileInputMode,
+} from "./types";
+import {
+  formatBytes,
+  getDefaultHelpers,
+  validateFileSize,
+  validateFileType,
+} from "./utils";
+import { FileService } from "./file.service";
+
+@Component({
+  selector: "tedi-file-dropzone",
+  templateUrl: "./file-dropzone.component.html",
+  styleUrls: ["./file-dropzone.react.scss", "./file-dropzone.component.scss"],
+  imports: [
+    IconComponent,
+    FeedbackTextComponent,
+    CardComponent,
+    CardContentComponent,
+    ClosingButtonComponent,
+    TooltipComponent,
+    TooltipTriggerComponent,
+    TooltipContentComponent,
+    InfoButtonComponent,
+    ReactiveFormsModule,
+    VerticalSpacingDirective,
+    TediTranslationPipe,
+  ],
+  providers: [FileService],
+})
+export class FileDropzoneComponent implements ControlValueAccessor, OnInit {
+  /**
+   * Specifies the allowed file types (e.g., "image/png, image/jpeg").
+   *
+   * Does not validate the contents of the file, only the file extension.
+   *
+   * https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Attributes/accept
+   **/
+  accept = input<string>("");
+  /**
+   * The maximum file size allowed for upload, in bytes.
+   * @default 0 (no limit)
+   **/
+  maxSize = input<number>(0);
+  /**
+   * Determines if multiple files can be uploaded at once via the file picker.
+   * @default false
+   **/
+  multiple = input<boolean>(false);
+  /**
+   * If true, each file is validated separately instead of rejecting all at once.
+    8 @default false
+   **/
+  validateIndividually = input(false);
+  /**
+   * An array of default files that are preloaded in the upload list.
+   * @default false
+   **/
+  defaultFiles = input<File[] | FileDropzone[]>([]);
+  /**
+   * The unique identifier for the input element that this label is associated with.
+   * This ID should match the input element's `id` attribute to ensure accessibility.
+   */
+  id = input<string>();
```

Comment:
think this naming should be changed to ala inputId or smth, otherwise the id attribute will be bound to the angular component and native input both